/ball_track:
  ros__parameters:
    mode: cv
    ball_type: orange
    image_topic: /zed/image_raw
    color_image_topic: /zed/image_raw
    depth_image_topic: /zed/depth_image
    camera_info_topic: /zed/camera_info
    model: ball-detect-hr.pt
    camera_frame: zed_camera_link